src/kubernetes_cluster/spec/controller/controller_runtime.rs

```diff
@@ -11,36 +11,113 @@ use builtin_macros::*;
 
 verus! {
 
-/// This action specifies how the watcher triggers reconcile.
-/// It is highly simplified compared to the actual watcher implementation in kube-rs:
-/// (1) The triggering condition should come from the developer. Currently we hardcode it in
-/// reconcile_trigger.
-/// (2) The watcher and reconciler run concurrently: the watcher can take incoming events while
-/// the reconciler is in reconcile. If the reconciler is working on object X, all the incoming
-/// events related to X will stay pending in a queue.
-/// (3) The watcher deduplicates triggering events for the same object: if an event for object X
-/// comes but there is already another pending event for the same object, the incoming event is
-/// discarded.
+pub open spec fn issue_initial_list<T>(reconciler: Reconciler<T>) -> ControllerAction<T> {
```

Comment:
Where is this used?

Reply:
I forgot to add the new actions to the state machine.
They are used here: https://github.com/vmware-research/verifiable-controllers/pull/71/files#diff-5a79ed1c20cfc0ea5794df6915706fa337a25076d5c905da54d1fe9cc56f6235R40